### PR TITLE
f-popover@2.2.0 and other - Elevation tokens values instead of direct box-shadow values

### PR DESCRIPTION
**Pull request by dandel10n**
### Changed
- box-shadow value to use the elevation token from the pie-design-tokens.

### CODE REVIEW

File: packages/components/molecules/f-promotions-showcase/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
+v0.4.0
+------------------------------
+*November 11, 2021*
```
Comment (simonsandersje) [inline packages/components/molecules/f-promotions-showcase/CHANGELOG.md]: February

File: packages/components/organisms/f-content-cards/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
+v7.1.0
+------------------------------
+*November 11, 2021*
```
Comment (simonsandersje) [inline packages/components/organisms/f-content-cards/CHANGELOG.md]: February

Comment (simonsandersje) [reply packages/components/organisms/f-content-cards/CHANGELOG.md]: 2022

File: packages/components/molecules/f-promotions-showcase/CHANGELOG.md (hunk 1 above)
Comment (simonsandersje) [inline packages/components/molecules/f-promotions-showcase/CHANGELOG.md]: 2022

**Review by DamianMullins — APPROVED** [review]

**Review by simonsandersje — APPROVED** [review]

**Pull request merged**